pkg/qliksense/context_configs.go

```diff
@@ -230,6 +230,21 @@ func (q *Qliksense) SetContextConfig(args []string) error {
 	return nil
 }
 
+func (q *Qliksense) ListContextConfigs() error {
+	qliksenseConfigFile := filepath.Join(q.QliksenseHome, QliksenseConfigFile)
+	var qliksenseConfig api.QliksenseConfig
+	api.ReadFromFile(&qliksenseConfig, qliksenseConfigFile)
+	if len(qliksenseConfig.Spec.Contexts) > 0 {
+		for _, cont := range qliksenseConfig.Spec.Contexts {
+			fmt.Println("Context Name : ", cont.Name, " CR File Location: ", cont.CrFile)
+		}
+		fmt.Println("Current Context : ", qliksenseConfig.Spec.CurrentContext)
+	} else {
```

Comment:
Will be great if the results are displayed better, as discussed.

Reply:
New view of results:
<img width="769" alt="Screen Shot 2020-02-19 at 11 48 04 AM" src="https://user-images.githubusercontent.com/24929514/74854962-04005680-530e-11ea-9402-0e571307097e.png">